### DISCUSSION TITLE
v1.9.0-beta1

### DISCUSSION THREAD

**Discussion by kyleconroy** [sqlc-dev/sqlc#1104] [Q&A]
Beta release of v1.9.0. Please report any issues here.

<hr /><em>This discussion was created from the release <a href='https://github.com/kyleconroy/sqlc/releases/tag/v1.9.0-beta1'>v1.9.0-beta1</a>.</em>

**Comment by johanbrandhorst** [comment]
Is there a change log available 😁?

**Reply by kyleconroy** [reply]
Not yet! There will be one for the final release.
